dll/win32/advapi32/misc/evntrace.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     advapi32.dll Event tracing stubs
+ * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include <wmistr.h>
+#include <evntrace.h>
+
+WINE_DEFAULT_DEBUG_CHANNEL(advapi);
+
+
+TRACEHANDLE
+WINAPI
+OpenTraceA(IN PEVENT_TRACE_LOGFILEA Logfile)
+{
+    UNIMPLEMENTED;
+    return (TRACEHANDLE)INVALID_PROCESSTRACE_HANDLE;
```

Comment:
This shouldn't need a cast.

Reply:
It shouldnt, but GCC disagrees.

Reply:
Then something must be wrong. The definition literally includes that same cast.

Reply:
`evntrace.c:20:12: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]`

Reply:
Yeah, that's because INVALID_HANDLE_VALUE is a pointer and ULONG64 is a differently-sized integer. We should fix the definition to be `(TRACEHANDLE)(ULONG_PTR)INVALID_HANDLE_VALUE`.
I don't see how your cast here would be able to fix that warning though.

Reply:
I built again with gcc, and it was going for a few minutes so I assumed it was fine, but guess the order modules are built in is random.